studio/settings.py

```diff
@@ -403,62 +408,62 @@
+                "WARNING": "yellow",
+                "ERROR": "red",
+                "CRITICAL": "bold_red",
+            },
+            "format": "%(log_color)s%(asctime)s - %(levelname)s - %(name)s: %(message)s%(reset)s",
         },
     },
     "handlers": {
         "console": {
             "class": "logging.StreamHandler",
             "formatter": "colored",
         },
-        "json_file": {
-            "class": "logging.handlers.WatchedFileHandler",
-            "filename": "logs/json.log",
+        "json": {
+            "class": "logging.StreamHandler",
             "formatter": "json_formatter",
         },
-        "flat_line_file": {
-            "class": "logging.handlers.WatchedFileHandler",
-            "filename": "logs/flat_line.log",
-            "formatter": "key_value",
-        },
     },
     "loggers": {
-        "django_structlog": {
-            "handlers": ["json_file"],
-            "level": "DEBUG" if DEBUG else "WARNING",
-        },
-        # Make sure to replace the following logger's name for yours
-        "portal": {
-            "handlers": ["console"],
-            "level": "DEBUG" if DEBUG else "WARNING",
+        "": {
+            "handlers": ["console" if DEBUG else "json"],
+            "level": "DEBUG" if DEBUG else "INFO",
         },
+        'django.server': {
+            'handlers': ['console'],
+            'level': "DEBUG" if DEBUG else "INFO",
+            'propagate': False,
+        }
     },
 }
-
-structlog.configure(
-    processors=[
-        structlog.contextvars.merge_contextvars,
-        structlog.stdlib.filter_by_level,
-        structlog.processors.TimeStamper(fmt="iso"),
-        structlog.stdlib.add_logger_name,
-        structlog.stdlib.add_log_level,
-        structlog.stdlib.PositionalArgumentsFormatter(),
-        structlog.processors.StackInfoRenderer(),
-        structlog.processors.format_exc_info,
-        structlog.processors.UnicodeDecoder(),
-        structlog.stdlib.ProcessorFormatter.wrap_for_formatter,
-    ],
-    logger_factory=structlog.stdlib.LoggerFactory(),
-    cache_logger_on_first_use=True,
-)
+if not DEBUG:
+    structlog.configure(
+        processors=[
+            structlog.contextvars.merge_contextvars,
+            structlog.stdlib.filter_by_level,
+            structlog.processors.TimeStamper(fmt="iso"),
+            structlog.stdlib.add_logger_name,
+            structlog.stdlib.add_log_level,
+            structlog.stdlib.PositionalArgumentsFormatter(),
+            structlog.processors.StackInfoRenderer(),
+            structlog.processors.format_exc_info,
+            structlog.processors.UnicodeDecoder(),
+            structlog.stdlib.ProcessorFormatter.wrap_for_formatter,
+        ],
+        logger_factory=structlog.stdlib.LoggerFactory(),
+        cache_logger_on_first_use=True,
+    )
+
+
+def GET_LOGGER(__name__: str):
+    if DEBUG:
+        return logging.getLogger(__name__)
+    else:
+        return structlog.getLogger(__name__)
```

Comment:
Why did you go for this style and not like this?

```suggestion
def get_logger(name: str):
    if DEBUG:
        return logging.getLogger(name)
    else:
        return structlog.getLogger(name)
```

One might shoot themself in the foot by using module level variables for arguments, like `__name__`, I think

And function is not constant. So it's misleading to use capital letters for the name:)

Reply:
I agree about the `__name__` thing. But in order to define the function in `settings.py`, it must be capital letters 

> There’s nothing stopping you from creating your own settings, for your own Django apps, but follow these guidelines:
> 
> - Setting names must be all uppercase.
> - Don’t reinvent an already-existing setting.
 
 But i can create a file in studio called `utils.py` and put it there with a better name?